packages/editor/src/core/components/Cell/ErrorCell/index.tsx

```diff
@@ -1,24 +1,27 @@
 import React from 'react';
-import { useIsEditMode, useRemoveCell } from '../../hooks';
+import { useIsEditMode, useRemoveCell, useUiTranslator } from '../../hooks';
 
 const ErrorCell: React.FC<{ nodeId: string; error: Error }> = ({
   nodeId,
   error,
 }) => {
   const isEditMode = useIsEditMode();
   const removeCell = useRemoveCell(nodeId);
+  const { t } = useUiTranslator();
   return (
     <div className="react-page-cell-error">
-      <strong>An error occurred!</strong>
+      <strong>{t('An error occurred!') || 'An error occurred!'}</strong>
       <small>
         <dl>
-          <dt>Cause:</dt>
+          <dt>{t('Cause:') || 'Cause:'}</dt>
           <dd>{error.message}</dd>
-          <dt>Cell:</dt>
+          <dt>{t('Cell:') || 'Cell:'}</dt>
```

Comment:
"don't repeat yourself"

we should remove the `|| 'text'` here. instead the t-function handles the cases where the translator function does return null (or an empty string).

Remember that we control the t function that is returned from `useUiTranslator`.

Reply:
As we discussed before, the reason I added these `|| 'text'` is to show how we can separate `key` with labels. For example, if in the future, you need to change `Cell`, keeping the key untouched but add this instead. 

However, the default `(label) => label` defeat the purpose, we actually want to change the default `t` function to return `undefined` and hard code default labels in the code, just to separate keys with labels. Keys will never be changed, but labels could be changed. 

If you want to keep it simple, then default return keys as is would be a lot simpler, with fewer flexibilities.

We don't control the `t` function, we call it and use whatever been returned. Developers control this function.

Please let me know what you think of above.

Reply:
> We don't control the t function, we call it and use whatever been returned. Developers control this function.

no, developers control the UiTranslator function.

I just really don't like to have now every string twice now in the code. It just adds noise to the code.

The t function should return a default translation if the developers UiTranslator returns null.

What about this proposel:

the `t`-functon (that geturns by the hook) has this implementation:

```
const t = (key: string) => {
 const translated = userDefinedUiTranslatorFunction(key)
 if(translated === null){
   return defaultTranslatorFunction(key) // that can be the identity to begin with and later can use some default translation object
 }
}
```

so it would always work, even if the developer forgets to translate something